### DISCUSSION TITLE
Count data on drafts (cache types / log types)

### DISCUSSION THREAD

**Discussion by RubyShoos** [2Abendsegler/GClh#2086] [Ideas]
One of the things I am already missing most about the old Field Notes / Drafts is the count totals at the bottom of the list, which allowed me to see how many EarthCaches, Wherigos and Multis were waiting in my logging queue. And how many of the headline number were DNFs and Write Notes. Is there any way this feature can be redeveloped on the new Drafts page?

Also, I would really like a icon of the log type (Found It / DNF / Write Note) on each of the drafts in the list, like in the example. I don't know if this is possible either.

While I'm here, can I just say thank you for all of the great features of GClh - I cannot function without them.

![Example](https://user-images.githubusercontent.com/101502393/158055280-e5bcf3e3-110c-4854-a363-4e0c60313eb6.png)

**Comment by 2Abendsegler** [comment]
> thank you for all of the great features of GClh - I cannot function without them.

Thank you. 😊

All of your points make sense to me too. We'll include the points for one of the next versions.
- Statistic of cache types and log types
- Log type icon (maybe also as an overlaid icon over the cache type) 

Issue #2089

**Reply by RubyShoos** [reply]
Brilliant - I will look forward to it :-)

**Reply by RubyShoos** [reply]
Thank you for getting the icon feature developed so impressively swiftly! My drafts queue is often longer than I'd like it to be, so anything that helps me stay on top of where I am helps. #2095 

Really appreciated :-)

**Comment by capoaira** [comment]
I've prepared the feature in the PR #2108

You can install the temporary version here: https://github.com/capoaira/GClh/blob/drafts-stats/gc_little_helper_II.user.js?raw=true
Here it goes to the current stable release: https://github.com/2Abendsegler/GClh/blob/master/gc_little_helper_II.user.js?raw=true

To get the stats you have to click the button "Count cache and log types"

It may be taken some time for testing because we haven't many caches to test.

**Comment by RubyShoos** [comment]
Just looking back at this and wanted to drop a massive thanks for getting this feature back in play. It helps me live my life more efficiently and joyfully :-)
